### PR TITLE
add spanish localization

### PR DESCRIPTION
**Pull request by Jakayus**
Add Spanish localization to project. 

Closed issue #135 

## What it Does
This adds Spanish localization to the app.

## How I Tested
1. Edit Scheme language to Spanish 
2. Ran the BrainMarks app in Simulator and reviewed Spanish translations

## Notes

1. The settings view was recently updated with section headers, so this was added for Spanish localization.  This will have to be updated for the other languages.
 
2. The names of the new app icons are  in English. I wasn't sure if those should be in Spanish as well or not, but I also did not figure out how to change them even if I wanted to. 

3. Although there is a 'Latin America' subset of Spanish localization available (es-419), I do not think that it was needed for this app.  I ran the simulator with es-419 (vs es) and verified Spanish settings were still all there. 

## Screenshot
Added a screenshot of the settings view with Spanish translations in use.

Simulator Screen Shot - iPhone 14 Pro - 2022-10-18 at 22 21 59

### CODE REVIEW

**Comment by Jakayus** [comment]
yikes! found a (super small) issue after the PR was made.

TLDR;  "Settings" defined differently in two spots (lines 14/44), but app uses the later one (correct translation), so issue is not seen visually in the app.

I'm guessing we can just fix the typo on line 11 or perhaps get ride of the second settings? It appears that it takes the latest translation in a localization file for any duplicate translations. (I made a second "Tweets" at the bottom and gave it a random name which then got used, so I believe the latest of the duplicates is used for the app).

Regarding the PR - if I want to make an additional change, how does that work with an open PR? Do I need to re-squash or do anything special if any new commits are made?

**Comment by mikaelacaron** [comment]
I'm not able to check the issue you mentioned, but if you want to add stuff go ahead and fix it and push to your branch (the PR updates automatically)

(There will be 2 commits now rather than one, which is fine)

But if you push more changes (like 3 or more commits) can you re-squash them? [you'll have to force push, if you squash again]

**Comment by Jakayus** [comment]
Thanks for that info!

No more planned commits, should be good to go.

**Review by mikaelacaron — APPROVED** [review]
@Jakayus Great work!!
